### PR TITLE
[AFD - Maybe future PR?] Hypnodisc

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
If you aren't British or had no childhood, don't read this PR.

Adds Hypnodisc as a 70 TC hijack-only vehicle. Instead of being remote control, its a ride-on vehicle where colliding with a human will deal 100 damage to it. Yes its a lot of damage, but its a 70TC hijack item that sec can shoot you off of.

I wanted to make a stats listing popup with all the data from the bot but I couldnt be bothered. 

## Why It's Good For The Game
I miss robot wars. Battlebots doesn't hold a candle to it. Give me Craig Charles back.

## Images of changes
See IDB and testing section.

**Yes I know its a hastily done sprite, much thanks to Venuska1117. This is an AFD PR and I dont care enough. Atleast the disc spins.**

## Testing
dreamseeker_SJoLR86wuC

## Changelog
:cl: AffectedArc07
add: Added Hypnodisc as a 70TC hijack only item. Use with caution.
/:cl:

### CODE REVIEW

File: code/modules/vehicle/hypnodisc.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	if(!istype(M, /mob/living/carbon/human))
+		// No murdering non humans
+		return
+
+	var/mob/living/carbon/human/H = M
+
+	H.adjustBruteLoss(75) // Youve been hit with a bigass spinning disc - its going to hurt
+	// We ignore canstun here, need to knock people over to avoid chain damage
+	H.Stun(100, TRUE)
+	H.Weaken(100, TRUE)
```
Comment (Contrabang) [inline code/modules/vehicle/hypnodisc.dm]:
```suggestion
	H.Weaken(10 SECONDS, TRUE)
```

File: code/modules/vehicle/hypnodisc.dm [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+	if(!istype(M, /mob/living/carbon/human))
+		// No murdering non humans
+		return
+
+	var/mob/living/carbon/human/H = M
+
+	H.adjustBruteLoss(75) // Youve been hit with a bigass spinning disc - its going to hurt
+	// We ignore canstun here, need to knock people over to avoid chain damage
+	H.Stun(100, TRUE)
```
Comment (Contrabang) [inline code/modules/vehicle/hypnodisc.dm]:
```suggestion
	H.Stun(10 SECONDS, TRUE)
```

**Comment by LouiseIsntHome** [comment]
11/10 you redeemed yourself

**Review by Contrabang — APPROVED** [review]

File: code/modules/vehicle/hypnodisc.dm [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	// Handle doors
+	..()
+
+	if(!has_buckled_mobs())
+		// No murder if no one is on it
+		return
+
+	if(!istype(M, /mob/living/carbon/human))
+		// No murdering non humans
+		return
```
Comment (GDNgit) [inline code/modules/vehicle/hypnodisc.dm:37]:
```suggestion

	if(!istype(M, /mob/living/carbon/human))
		// No murdering non humans
		return
```
LET ME USE THIS LIKE A BATTERING RAM DAMN IT

Comment (AffectedArc07) [reply code/modules/vehicle/hypnodisc.dm]: Ok

**Comment by AffectedArc07** [comment]
At request of GDN

disctruction

**Pull request merged**